interface/reports/pat_ledger.php

```diff
@@ -459,7 +459,7 @@ function sel_patient() {
         $arrOeUiSettings = array(
         'heading_title' => xl('Report') . " - " . xl('Patient Ledger by Date'),
```

Comment:
On line 185 to 186, convert to use style background-color instead of bgcolor

Reply:
Also on lines 279 to 280, convert to use style background-color instead of bgcolor